Revit_Core_Engine/Create/View/ViewReflectedCeilingPlan.cs

```diff
@@ -76,7 +76,7 @@ public static View ViewReflectedCeilingPlan(this Document document, Autodesk.Rev
                     BH.Engine.Base.Compute.RecordWarning("Could not apply the View Template of Id " + viewTemplateId + "'." + ". Please check if it's a valid ElementId.");
```

Comment:
Before setting this property, we should obtain the element at the view template ID and check if it's an actual view template.